src/System.Threading.Thread/tests/ThreadTests.netstandard.cs

```diff
@@ -20,10 +20,21 @@ public static class ThreadTests
         [Fact]
         public static void ConstructorTest()
         {
-            var t = new Thread(() => { });
-            t = new Thread(() => { }, 0);
-            t = new Thread(state => { });
-            t = new Thread(state => { }, 0);
+            Action<Thread> startThreadAndJoin =
+                t =>
+                {
+                    t.IsBackground = true;
+                    t.Start();
+                    t.Join();
+                };
+            startThreadAndJoin(new Thread(() => { }));
+            startThreadAndJoin(new Thread(() => { }, 0));
+            startThreadAndJoin(new Thread(() => { }, 1));
+            startThreadAndJoin(new Thread(() => { }, 16 << 20));
```

Comment:
Maybe make this a `[Theory]` and pass in sizes, with the invalid and default cases separate tests?